### PR TITLE
Added new layout

### PR DESCRIPTION
**Pull request by ar4s**
Connected with https://github.com/allegro/ralph/pull/2166

### CODE REVIEW

**Comment by szok** [comment]
:ok:

**Comment by mkurek** [comment]
please add `--find-links=https://pypi.python.org/pypi/pysphere/0.1.8` in .travis.yml to ralph installation (like in https://github.com/allegro/ralph/blob/develop/Makefile)

**Comment by mkurek** [comment]
:ok:

File: src/ralph_assets/models_assets.py [hunk 1]
```diff
@@ -252,6 +252,7 @@ class ModelVisualizationLayout(Choices):
     na = _('N/A')
     layout_1x2 = _('1x2').extra(css_class='rows-1 cols-2')
     layout_2x8 = _('2x8').extra(css_class='rows-2 cols-8')
+    layout_1x14 = _('1x14').extra(css_class='rows-1 cols-14')
```
Comment (mkurek) [inline src/ralph_assets/models_assets.py]: move it to the end

**Comment by mkurek** [comment]
:ok:

**Pull request merged**